Organising covid edition (#44)
lrug/lrug.github.io: organising/organising-a-meeting.md

---
layout: default
title: Organising an LRUG meeting
---

Part of our [organising](/organising) documentation.

# Organising a meeting

> Note that this documents our current procedures during the COVID-19 pandemic where we're running remote meetings.  It'll be slightly different when we return to in-person meetings.

> Someone needs to:

Reviewer: ```suggestion
We need to:
```

### A full agenda

We run the meetings from 6:30pm to 8:00pm.  With 10 minutes or so admin at the start, that means we have about 1 hour and 20 minutes (80 minutes) to fill.  Combinations that work well:

* one 10 minute talk, one 25 minute talk, one 40 minute talk (~75 minutes)
* three 25 minute talks (~75 minutes)
* two 25 minute slots, two 10 minute slots (~70 minutes)

Combinations that work, but are painful:

* two 40 minute talks (~80 minutes) - 40 minute talks can drag on a bit, we try to avoid this, but sometimes this is what we

Special combinations:

* eight 10 minute talks (~80 minutes) - usually this is what we do for our February lightning talks only event.  It's a fun evening, but tough to organise so we don't do it that often.  If you have a glut of speakers though, by all means!
* one 80 minute session - for a practical event, or panel discussion, or pub quiz, or something.  We wouldn't want a single speaker to talk for 80 minutes, but if someone has an idea for a interactive session that would take up the whole time then why not try it!

> But!  Any combination that fills roughly 80 minutes will do.  It's also fine to not fill 80 minutes.  Getting one speaker in just so we can make an LRUG happen is good.

Reviewer: I think we should move this line up nearer the top -- it's very rare at the moment that we "fill" the time, and I wouldn't want anyone new feeling like they should expect to. The "combinations that work well" a few paragraphs up are more the exception than the rule (but still useful to include).

Speakers often don't fill their allotted time, so we frequently end before our self-imposed 8pm curfew.

### What if it doesn't go smoothly

Hopefully all that is needed is for you to pick the top 2 or 3 volunteers who are already earmarked for the meeting you are organising and email them to confirm the date and their timeslot.  They get back to you and the meting is full of speakers.

It doesn't always go that smoothly.

* Speakers pull out
* Speakers just don't respond
* We don't have anyone earmarked for that meeting

> In this case you'll need to actually find some speakers as well as confirm them.  Check out [our tips on populating the speaker backlog](/organising/maintaining-the-community#populate-the-speaker-backlog) if you need to find speakers.  Adding an urgent "_for this month's meeting_" plea can help.

Reviewer: ```suggestion
In this case we'll need to actually find some speakers as well as confirm them.  Check out [our tips on populating the speaker backlog](/organising/maintaining-the-community#populate-the-speaker-backlog) if you need to find speakers.  Adding an urgent "_for this month's meeting_" plea can help.
```

## Announce the meeting

> Once you have at least one speaker confirmed you are ready to announce the meeting.  This means:

Reviewer: ```suggestion
Once we have at least one speaker confirmed, then we are ready to announce the meeting.  This means:
```

1. [Set up the eventbrite page so people get reserve their place](/organising/organising-a-meeting#set-up-the-eventbrite-page)
2. [Schedule the zoom meeting](/organising/organising-a-meeting#schedule-the-zoom-meeting)
3. [Write up the meeting on lrug.org](/organising/organising-a-meeting#write-up-on-lrugorg)
4. [Tell the mailing list about it](/organising/organising-a-meeting#tell-the-mailing-list-about-the-meeting)
5. [Tweet about it](/organising/organising-a-meeting#tweet-about-the-meeting)

Ideally we do all this the week after the previous meeting, but sometimes we do it really close to the event if we're struggling to get speakers.  It can be useful to write up placeholder versions and publish them, so that people will at least get the event into their calendars.

> If you have more than one speaker it can be useful to trickle the speaker announcements out over a couple of weeks in order to keep reminding people.

Reviewer: ```suggestion
If we have more than one speaker it can be useful to trickle the speaker announcements out over a couple of weeks in order to keep reminding people.
```

### Set up the eventbrite page

We use eventbrite to collect attendee details and manage how many people can come.  Our zoom account has 100 places so we limit the eventbrite page to 95 tickets to leave room for the host and speakers.

We like to be creative with this page, add a fun header image and create amusing ticket types.  We tend not to put full meeting details into this and just point people at lrug.org where the write up is.

* set ticket sales to close at 5:30pm on the day
* limit total ticket sales to 95
* add a link to the zoom meeting as a plain link to the "online event page" - we haven't linked our zoom account, and the link works just fine.
* check that there are scheduled emails to attendees on the day to send the link out to them before 6:30pm
* enable the waiting list in the off chance the meeting "sells out"

### Schedule the zoom meeting

We schedule the zoom meeting in advance so we can:

1. add the link to the "online event page" in eventbrite
2. pass on the link to the speakers well in advance

> 3. get the recording properly named for the meeting

Reviewer: Maybe move this section above the Eventbrite one, so that the zoom meeting link is already available for use there?

### Write up on lrug.org

The write-ups are fairly straightforward, so much so you can copy the previous meeting page and replace the details.

> 1. Change the registration url to the new eventbrite page

Reviewer: ```suggestion
1. Change the registration url to the new eventbrite page. Make sure you also change the anchor link destination - we typically have a link in the meeting details along the lines of `[register here](#sept21registration)`, which ensures that the link jumps to the right registration block even when we're displaying multiple meeting details on the same page (e.g. the index)
```

3. Update the frontmatter to include `created_at` / `published_at` timestamps that are _broadly_ correct (they're used to order the pages)
4. Make sure the date and times are correct for the announcement
5. If you feel up to it tweak some of the wording to make the announcement unique, but this is far from a requirement - after all there are only so many way to say "please register", "hang out afterwards", "here are the talks"

Push this up as a PR to [the lrug.org github repo][github-lrug.org] and once CI has okayed it you can merge and this will automatically deploy to lrug.org.

### Tell the mailing list about the meeting

Once the meeting is written up on lrug.org you should email [the mailing list][mailing-list] to tell people about it.  We keep this short and to the point:

1. Include the date and start and end times
2. Include titles and speaker names
3. Point people at lrug.org to find out more
4. Point people at the readme to agree to the code of conduct
5. Point people at eventbrite to register for a place
6. Ask for more volunteers - even if the agenda is full, it can be a good hook for future volunteers

### Tweet about the meeting

> Once the meeting is written up on lrug.org you should tweet about it.  Twitter has even less space than email, so keep it brief and point people at lrug.org for full details.  Feel free to use threads, but no-one wants a 30-tweet storm to announce a meeting.

Reviewer: ```suggestion
Once the meeting is written up on lrug.org you should tweet about it.  Twitter has even less space than email, so keep it brief and point people at lrug.org for full details.  Feel free to use threads, but no-one wants a 30-tweet storm to announce a meeting.

If you don't have the credentials for Twitter, get in touch with another organiser and they will be happy to share them with you.
```

1. Mention the speakers by their twitter handles if you have them
2. Point people at lrug.org for full details and registration info

It can be useful to tweet a few times in the lead up, and to tweet on the morning of the actual meeting.  A lot of people need reminding and twitter is as good a way as any.

## Host the meeting

> This is quite involved, so [check out our separate "hosting the meeting" documentation](/organising/hosting-a-meeting) for all the detail.  The short version is that you need to:

Reviewer: ```suggestion
This is quite involved, so [check out our separate "hosting the meeting" documentation](/organising/hosting-a-meeting) for all the detail.

It's also important to remember that just because you're responsible for organising the meeting, it doesn't mean that you need to be the person hosting the meeting that month. Yours is the power to delegate!

The short version is that you need to:
```
